libs/designer/src/lib/core/utils/parameters/helper.ts

```diff
@@ -1730,6 +1731,7 @@ export async function loadDynamicValuesForParameter(
     if (isDynamicDataReadyToLoad(dependencyInfo)) {
       dispatch(
         updateNodeParameters({
+          isUserAction: false,
```

Comment:
can we actually just make this optional and treat undefined as false?

Reply:
Sure. I had gone the route of being explicit but I have no actual preference. Will make that change.